server/server.go

```diff
@@ -598,7 +598,7 @@ func (server *Server) loadConfig(configurations configs, globalConfiguration Glo
 				}
 
 				entryPoint := globalConfiguration.EntryPoints[entryPointName]
-				if entryPoint.Redirect != nil {
+				if entryPoint.Redirect != nil && configuration.Backends[frontend.Backend] == nil {
```

Comment:
This change seems to break everything.
Could you give us a working configuration to test this use case (toml)?